pkg/apis/extensions/validation/dnsrecord_test.go

```diff
@@ -104,30 +104,31 @@ var _ = Describe("DNSRecord validation tests", func() {
 			}))))
 		})
 
-		It("should forbid unsupported recordType values", func() {
```

Comment:
If I see it correctly, you removed the `should forbid unsupported recordType values` case without a proper replacement.
Can you re-introduce the case and use a different record type, that is not supported, e.g. `SOA`?